### ISSUE TITLE
Use migz for decompression of big gzipped files

### ISSUE THREAD

**Issue by aroelo** [The-Academic-Observatory/observatory-platform#69]
Gzipped files can be decompressed with standard gunzip, however there are more efficient alternatives that might speed up the process significantly for big files.

One alternative is migz (https://github.com/linkedin/migz), a java based tool that can decompress in parallel.

It is still a bit of a challenge to get such an external tool installed on every node in cloud composer.
See https://stackoverflow.com/questions/50840760/how-to-apt-get-install-packages-on-kubernetes-for-google-cloud-composer?rq=1 for a similar question.

I think I would have to install java, gradle, git clone the migz repo, and then install migz. It seems a bit much to do all in one task, but I will have a look.

**Comment by rhosking** [comment]
I suspect this work is on hold or blocked, @aroelo I have added a label to show this. Feel free to update as needed, or if you have any further thoughts leave them here

**Comment by rhosking** [comment]
@aroelo @jdddog @tuanchien is this still something we want to explore?

**Comment by jdddog** [comment]
Migz can only de-compress gzip files that were compressed by itself, so unfortunately it won't help for our use cases.

https://github.com/linkedin/migz/issues/5
